modules/ocf/manifests/firewall/pre.pp

```diff
@@ -0,0 +1,73 @@
+class ocf::firewall::pre {
+  $firewall_defaults = {
+    require => undef,
+  }
+
+  firewall {
+    default:
+      * => $firewall_defaults;
+    '000 accept all icmp':
+      chain  => 'PUPPET-INPUT',
+      proto  => 'icmp',
+      action => 'accept';
+
+    '000 accept all icmpv6':
+      chain    => 'PUPPET-INPUT',
+      proto    => 'ipv6-icmp',
+      action   => 'accept',
+      provider => 'ip6tables';
+  }
+
+  ocf::firewall::firewall46 {
+    default:
+      * => $firewall_defaults;
+
+    '001 allow RELATED and ESTABLISHED traffic':
+      opts => {
+        'chain'  => 'PUPPET-INPUT',
+        'proto'  => 'all',
+        'state'  => ['RELATED', 'ESTABLISHED'],
+        'action' => 'accept',
+      };
+  }
+
+  firewall {
+    default:
+      * => $firewall_defaults;
+
+    '002 allow ssh from desktops (IPv4)':
+      chain     => 'PUPPET-INPUT',
+      src_range => '169.229.226.100-169.229.226.139',
```

Comment:
We might want to make these ranges hiera variables, that way it's easier to change all of them at once